src/DynamoCore/Models/RecordableCommands.cs

```diff
@@ -109,18 +91,6 @@ internal XmlElement Serialize(XmlDocument document)
                 return element;
             }
 
-            /// <summary>
-            /// This method serializes the RecordableCommand object in the json form.
-            /// The resulting string contains command type name and all the
-            /// arguments that are required by this command.
-            /// </summary>
-            /// <returns>The string can be used for reconstructing RecordableCommand
-            /// using Deserialize method</returns>
-            internal string Serialize()
-            {
-                return JsonConvert.SerializeObject(this, jsonSettings);
```

Comment:
I thought these were used when crashes occur, and we generate a document with the last commands that were run.
Maybe that was using something else..

Reply:
we do use recordable commands for that, but we use the xml serialization and deserialization code path that I left in.